src/app/container/tool-file-editor/tool-file-editor.component.ts

```diff
@@ -113,7 +113,10 @@ export class ToolFileEditorComponent extends FileEditing {
           // Only stop editing when version change was successful (not 204)
           this.toggleEdit();
           this.containerService.setTool(editedDockstoreTool);
-          this.alertService.detailedSuccess();
+          const updatedVersion = this.getNewestVersion(editedDockstoreTool.workflowVersions);
+          this.alertService.detailedSuccess(
+            'Saved version ' + updatedVersion.name + ' ' + 'of hosted workflow ' + editedDockstoreTool.namespace
```

Comment:
Must fix: This is a message for a tool, not a workflow.

Optional: Trivial, but instead of `...+ ' ' + 'of hosted...`, you could just put the space directly in the second string, e.g., `+ ' of hosted`. Makes is a little easier to read. Or you could use string interpolation, as I believe I suggested in another PR, but that's a matter of preference.

Reply:
no problem! sorry about not catching those minor errors. fixing right now